### PR TITLE
data:SarthakNegi

### PR DESCRIPTION
**Pull request by sarthaksarthak9**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

You Guys are doing amazing work!! keep it up.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6636"></a>

### CODE REVIEW

File: data/SarthakNegi.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "name": "Contribute to Open Source in our GitHub Organisation",
+      "url": "https://github.com/sarthaksarthak9",
+      "icon": "FaGithub"
+    },
+
+    {
+      "name": "LinkedIn Organisation",
+      "url": "https://www.linkedin.com/in/sarthak-negi-4923a6259/",
+      "icon": "FaLinkedin"
+    },
```
Comment (khushalarora) [inline data/SarthakNegi.json:22]: Please remove the extra **'Comma'** from this line :+1: 
```Suggestion
    }
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @sarthaksarthak9, thank you for contributing :tada: 
Please rename your file as with your GitHub username **`sarthaksarthak9`**
It will look like **`data/sarthaksarthak9.json`**

File: data/SarthakNegi.json (hunk 1 above)
Comment (sarthaksarthak9) [reply data/SarthakNegi.json:22]: ok , thank you for reviewing !!

File: data/sarthaksarthak9.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Sarthak Negi",
+  "type": "personal",
+  "bio": "devOps enthusiast , learning webd , open source advocate!",
+  "tags": ["Sarthak", "Open Source", "GitHub", "Community", "Collaboration"],
+  "links": [
+    {
+      "name": "Follow me in twitter and linkedin",
```
Comment (kumarsonsoff3) [inline data/sarthaksarthak9.json:8]: It's twitter link here, so shouldn't it be just like `Follow me on Twitter`. 
Rest looks good to me! 

```suggestion
      "name": "Follow me on Twitter",
```

Comment (sarthaksarthak9) [reply data/sarthaksarthak9.json:8]: I'll do it!!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉 

<p align="center"><a href="https://linkfree.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**